src/database/schemas/users.ts

```diff
@@ -0,0 +1,7 @@
+import mongoose from "mongoose";
+
+export const usersSchema = new mongoose.Schema({
+  fullName: { type: String, required: true },
```

Comment:
We should keep database columns atomic - so the fullName = firstName + lastName.